orchestrator-salt/src/main/resources/salt/salt/log4jpatch/scripts/cm_cdp_cdh_log4j_jndi_removal.sh

```diff
@@ -0,0 +1,289 @@
+#!/bin/bash
```

Comment:
Can we copy over the scripts as it is?  As I see you have concatenated into one file but important parts like pipefail are missing.

Reply:
there is no pipefail in cm_cdp_cdh_log4j_jndi_removal.sh, only in run_log4j_patcher, so I copied all of the content of cm_cdp_cdh_log4j_jndi_removal

Reply:
I have added run_log4j_patcher.sh also to the PR